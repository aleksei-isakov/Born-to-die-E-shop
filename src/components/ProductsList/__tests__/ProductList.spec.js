import { shallowMount } from '@vue/test-utils';
import ProductList from '@/components/ProductsList/ProductsList.vue';

describe('ProductList', () => {
  it('renders correctly', () => {
    const wrapper = shallowMount(ProductList);

    expect(wrapper.element).toMatchSnapshot();
  });
});
