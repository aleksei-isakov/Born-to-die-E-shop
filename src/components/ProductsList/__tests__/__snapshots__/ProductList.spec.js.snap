// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProductList renders correctly 1`] = `
<div
  class="products-list"
>
  <recentlyadded-stub
    text="Recently added"
  />
   
  <ul
    class="products-list__items"
  >
    <productitem-stub
      description="[object Object]"
      id="01ea1d1e-003d-465e-90c1-e5d3c521a618"
      price="800.7263828"
    />
    <productitem-stub
      description="[object Object]"
      id="72b9e64c-d056-4c8a-85c0-0e1175138ffd"
      price="800"
    />
    <productitem-stub
      description="[object Object]"
      id="2021-11-14T10:15:25.926Z"
      price="800"
    />
  </ul>
</div>
`;
